### PR TITLE
added link to use hompage from dahlem.html.

### PR DESCRIPTION
**Pull request by codekhal**

### CODE REVIEW

File: _layouts/overview.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
     <link href="css/deactivate-offline.css" rel="stylesheet"/>
     <link href="{{ site.local_server }}/offline/activate_localhost.css" rel="stylesheet"/>
     <link href="offline/activate.css" rel="stylesheet"/>
+    <a link href="dahlem.html"> homepage </a>
```
Comment (niccokunzmann) [inline _layouts/overview.html]: This will add a link to dahlem to the potsdam.html file.
I am not sure what the purpose is. Could you explain?

Comment (niccokunzmann) [reply _layouts/overview.html]: `a` links should be placed inside the body.

Do you speak English?
Sprichst du Deutsch?
Ты говоришь по-русский?

Comment (codekhal) [reply _layouts/overview.html]: I  understand English. My main purpose of adding this was linking two pages namely  dahlem.html to the potsdam.html file and i am not sure i by mistake added them in the header. Inconvenience is regretted. As I am a beginner hope to work on your repository  in future.

Comment (niccokunzmann) [reply _layouts/overview.html]: Hi, sorry I read this only now. I somehow missed the notification.
Would you like to update repository to the latest changes and have a look? Somebody else added a homepage link.

The idea is to have multiple pages of dojos which are independent of each other. The index file contains the overview of all pages and the dojo pages do not need to link to each other. They can do it after the second `---` if they like to.

Is there any issue you would like to contribute to?
You can browse the page and create an issue for what you think should be changed. 
If you have questions, feel free to ask.

Comment (niccokunzmann) [reply _layouts/overview.html]: What would you like to do?

**Comment by niccokunzmann** [comment]
Thanks for your contribution. I will only merge, if I can talk to you. Please let us know if you are interested in contributing in the future.

File: _layouts/overview.html (hunk 1 above)
Comment (codekhal) [reply _layouts/overview.html]: Sorry for replying late. I would like to contribute in your repository for other issues thanks for considering my pull request.

**Pull request closed without merging**